Games/types/Mafia/roles/cards/BanquetInvite.js

```diff
@@ -0,0 +1,41 @@
+const Card = require("../../Card");
+const { PRIORITY_BANQUET_MEETING } = require("../../const/Priority");
+
+module.exports = class BanquetInvite extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
```

Comment:
same as justice/ mistletoe, use the multi meeting function~
thank you

Reply:
done